src/eth_rpc/servers/eth_rpc.rs

```diff
@@ -235,9 +241,21 @@ where
         Err(EthApiError::Unsupported("eth_sendTransaction").into())
     }
 
+    #[allow(unreachable_code)]
     #[tracing::instrument(skip_all, ret, err)]
     async fn send_raw_transaction(&self, bytes: Bytes) -> Result<B256> {
         tracing::info!("Serving eth_sendRawTransaction");
+        #[cfg(feature = "rpc_forwarding")]
+        {
+            let provider_builded = ProviderBuilder::new().on_http(Url::parse(&MAIN_RPC_URL).expect("invalid rpc url"));
+
+            let tx_hash = provider_builded
+                .send_raw_transaction(&bytes)
+                .await
+                .map_err(|_| EthApiError::EthereumDataFormat(EthereumDataFormatError::TransactionConversion))?;
```

Comment:
I don't think we want to return this error, anything else we could return?

Reply:
I replaced it with a `.expect("failed to send transaction")`, I couldn't find a better error. What do you think?

Reply:
we don't want expects as this will cause the binary to panic. I think it's ok to use `TransactionError::BroadCasting`